docs/UserGuide.md

```diff
@@ -194,11 +213,13 @@ _Details coming soon ..._
 
 ## Command summary
 
-Action | Format, Examples
---------|------------------
-**Add** | `add n/<NAME> s/<STATUS> [t/<TAG>]…​` <br> e.g., `n/ReZero s/watching t/Anime`
-**Delete** | `delete <INDEX>`<br> e.g., `delete 3`
-**List** | `list`
-**Exit** | `exit`
+Action | Format, Examples                                                               
+--------|--------------------------------------------------------------------------------
+**Add** | `add n/<NAME> s/<STATUS> [t/<TAG>]…​` <br> e.g., `n/ReZero s/watching t/Anime` 
+**Delete** | `delete <INDEX>`<br> e.g., `delete 3`                                          
+**List** | `list`                                                                         
+**Exit** | `exit`                                                                         
+|     **Edit**    | `edit <INDEX> [n/<NAME>] [s/<STATUS>] [t/<TAG>]…​` <br> e.g., `n/ReZero s/watching t/Anime`                            |
```

Comment:
Should there be an additional | on the left of `**Edit**`?

Reply:
not sure why that appears but it looks fine 
![image](https://user-images.githubusercontent.com/77216013/158570272-5b8f5b54-61a6-4d4a-8a7e-24b4f6262418.png)

Reply:
Should the `Exit` and `Edit` be swapped in the table?

Reply:
If you want alphabetical order, i can swap

Reply:
Personally i feel that adding latest feature to the last row would be easier unless yall got ocd then we sort by alphabet.

Reply:
Ah I just found it weird that Edit's the only one with that `|` on the left.
Arranging by alphabetical order might help the reader when they're scanning through the commands available, but they could also `Ctrl + F`. I'd prefer alphabetical order though, just my 2 cents on this issue.

Reply:
Alright, i change it to alphabetical order, let me know if everything is still not to your satisfaction.